sys-devel/rust-std/rust-std-1.77.0.ebuild

```diff
@@ -0,0 +1,159 @@
+# Copyright 2020-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..12} )
+
+inherit flag-o-matic multiprocessing python-any-r1 rust-toolchain toolchain-funcs
+
+DESCRIPTION="Rust standard library, standalone (for crossdev)"
+HOMEPAGE="https://www.rust-lang.org"
+SRC_URI="https://static.rust-lang.org/dist/rustc-${PV}-src.tar.xz"
+
+LICENSE="|| ( MIT Apache-2.0 ) BSD-1 BSD-2 BSD-4"
+SLOT="stable/$(ver_cut 1-2)"
+# please do not keyword
+#KEYWORDS="" #nowarn
+IUSE="debug"
+
+BDEPEND="
+	${PYTHON_DEPS}
+	~dev-lang/rust-${PV}:=
+"
+
+DEPEND="||
+	(
+		>="${CATEGORY}"/gcc-4.7:*
+		>="${CATEGORY}"/clang-3.5:*
+	)
+"
+
+RDEPEND="${DEPEND}"
+
+# need full compiler to run tests
+RESTRICT="test"
+
+QA_FLAGS_IGNORED="usr/lib/rust/${PV}/rustlib/.*/lib/lib.*.so"
+
+PATCHES=(
+	"${FILESDIR}"/1.75.0-handle-vendored-sources.patch  # remove for >=1.77.0
+)
```

Comment:
Hello @bowlofeggs,
It looks like this patch isn't needed anymore.
```suggestion
```

Reply:
Thanks! Fixed in https://github.com/gentoo/gentoo/pull/35866/commits/87927be1014b8c8c24e1c40c7d8ba23c27086e7d